docs/en/status-communities/vote-to-make-a-community-private-or-public.md

```diff
@@ -0,0 +1,100 @@
+
+Voting involves five steps:
+
+- [x] [Connect your wallet][connect-wallet].
+- [x] [Initiate a voting process][initiate-vote].
+- [x] [Cast your vote][cast-vote].
+- [x] [Verify the vote][verify-vote].
+- [x] [Finalize the vote][finalize-vote].
+
+!!! note
+    Status does not charge you any fees for voting.
+
+!!! note
+    To know more about the dApp and the rules of each step, check out [About voting to change community visibility][about-voting-to-change-community-visibility].
+
+## Step 1: Connect your wallet {: #connect-wallet}
+
+Make sure your wallet contains enough SNT for voting and ETH for the gas fee.
+
+1. Open your wallet and switch the network to Optimism.
+1. Open the community directory curation dApp and select **Connect** from the top navigation.
+1. Select **Confirm** in your wallet.
+
+!!! note
+    You can use Status Wallet or MetaMask.
+
+## Step 2: Initiate a voting process {: #initiate-vote}
+
+Initiate a vote if there isn't an ongoing vote for the community you want to add or remove. Afterwards, other SNT holders can participate.
+
+### Initiate a vote for adding a community to Discover
+
+You need the [community's public key][import-a-community-someone-shares-with-you] to initiate a vote for adding a community.
+
+1. From the top navigation, select **Votes**.
+1. Select **Propose community**.
+1. Enter the community's public key.
+1. Choose the amount of SNT to stake and select **Confirm vote to add community**.
+1. Select **Create vote to add community**.
+1. Confirm the transaction and pay the gas fee from your wallet.
+
+### Initiate a vote for removing a community from Discover
+
+1. From the top navigation, select **Directory** to view all existing communities.
+1. Find the community you want to remove and select :desktop-delete: **Delete** next to the community name.
+1. Choose the amount of SNT you want to stake for voting and select **Confirm vote to remove community**.
+1. Confirm the transaction and pay the gas fee from your wallet.
+
+## Step 3: Cast your vote {: #cast-vote}
+
+Skip this step if you initiated the voting process. The voting period lasts for 14 days.
+
+1. From the top navigation, select **Votes** to view ongoing votes.
+1. Find the community you want to vote for. If it's a vote for adding a community, select **Don't add** or **Add**. If it's a vote for removing a community, select **Keep** or **Remove**.
+1. Choose the amount of SNT to stake.
+1. If it's a vote for adding a community, select **Vote to not add community** or **Vote to add community**. If it's a vote for removing a community, select **Vote to keep community** or **Vote to remove community**.
+
+## Step 4: Verify the vote {: #verify-vote}
+
+After voting, the 7-day verification period begins. During this period, valid votes are identified.
```

Comment:
We should emphasize this step is optional, but the more peers participate, the better it is for vote integrity.